tests/Message/FormatterTest.php

```diff
@@ -123,6 +123,16 @@ public function testFormatWithSetTimestampFormat(): void
         $this->assertSame($expected, $this->formatter->format($message, []));
     }
 
+    public function testFormatWithSetTimestampFormatWithCommaSeparator(): void
```

Comment:
Maybe separate long names like that - `testFormatWithSetTimestampFormat_WithCommaSeparator`? Very hard to read to be honest.

Reply:
This is volation of camelCase style.